### DISCUSSION TITLE
Pimcore Portal engine: can i Disable / Enable Login ?

### DISCUSSION THREAD

**Discussion by RassasAymen** [pimcore/pimcore#10900] [Q&A / Support]
there is a possibility to disable login from Portal engine Frontend and make it access free for everyone without login ?

**Comment by fashxp** [comment]
not possible yet. it is on our roadmap though.

**Reply by RassasAymen** [reply]
Thanks Christian for your reply.

**Comment by HendrikStone** [comment]
Hi @fashxp 
We also have a use case for this requirement. Do you already know when you want to roll it out?
Thanks for your help

**Reply by fashxp** [reply]
if you are enterprise customer, you can already check it out when installling the corresponding branch: 
`composer require pimcore/portal-engine:"dev-guest-login as 2.3"`

rollout planned after corresponding feedback and testing, in the course of Q2 planned.
